README.md

```diff
@@ -1,15 +0,0 @@
-# transloadit-js-client
```

Comment:
Would be cool if the project had *some* kind of readme still. I'm not against keeping the design goals where everyone can see them.